classes/development.js

```diff
@@ -0,0 +1,184 @@
+/* eslint-disable consistent-return */
+
+'use strict';
+
+const abslog = require('abslog');
+const rollup = require('rollup');
+const commonjs = require('rollup-plugin-commonjs');
+const resolve = require('rollup-plugin-node-resolve');
+const atImport = require('postcss-import');
+const { join } = require('path');
+const autoprefixer = require('autoprefixer');
+const postcss = require('postcss');
+const fs = require('fs');
+const cssnano = require('cssnano');
+const tempDir = require('temp-dir');
+const chokidar = require('chokidar');
+const { readMetaFile, writeMetaFile } = require('../utils');
+
+module.exports = class Meta {
+    constructor({ logger, name, watch, cwd, debug, js, css } = {}) {
+        this.log = abslog(logger);
+        this.watch = watch;
+        this.cwd = cwd;
+        this.debug = debug;
+        this.js = js;
+        this.css = css;
+        this.path = join(tempDir, `publish-${name}`, 'development');
+    }
+
+    async run() {
+        this.log.debug('Creating development asset bundles');
+        const development = {};
+        try {
```

Comment:
Is this try catch needed around the whole block? Its two levels of try catch here.

Reply:
I've now flattened it a bit